### PR TITLE
(ASC-1140) Added checking server creating status

### PR DESCRIPTION
**Pull request by phongdly**
System tests failed many tests in which a server needs to be created. During the server creation, the server status was seen to change from "Build" to "ERROR". It is unsure that there is at least one server has been successfully created, including the test_server created in server_setup.yml

This PR to add a step of checking if the server created in server_setup.yml is successful.

### CODE REVIEW

File: tasks/server_setup.yml [hunk 1]
```diff
@@ -10,3 +10,24 @@
...
     "{{ test_server }}"'
+
+- name: List test server
+  shell: |
+    lxc-attach -n "{{ utility_container.stdout }}"  \
+    -- bash -c '. /root/openrc ; \
+    openstack server list | grep "{{ test_server }}"'
+  register: test_server_instance
+  changed_when: false
+  failed_when: false
```
Comment (johnduarte) [inline tasks/server_setup.yml]: I don't believe that lines 20 and 21 are needed here. If the shell command fails to grep for the test server it will return an exit code of `1` which will naturally trigger a failure without these additional conditions.

```
$ ansible localhost -m shell -a "echo 'foo' | grep bar"

localhost | FAILED | rc=1 >>
non-zero return code
```

File: tasks/server_setup.yml [hunk 2]
```diff
@@ -10,3 +10,24 @@
...
+
+- name: Check test server status
+  # helpers.get_expected_value('server', instance_name, 'status', 'ACTIVE', host))
+  shell: |
+    lxc-attach -n "{{ utility_container.stdout }}"  \
+    -- bash -c ". /root/openrc ; \
+    openstack server show -f json \"{{ test_server }}\" | python -c 'import sys, json; print json.load(sys.stdin)[\"status\"]'"
```
Comment (johnduarte) [inline tasks/server_setup.yml]: The openstack cli offers an easier way to get the status value:
```
openstack server show -f value -c status \"{{ test_server }}\"
```

File: tasks/server_setup.yml (hunk 1 above)
Comment (johnduarte) [inline tasks/server_setup.yml]: I don't believe that lines 20 and 21 are needed here. If the shell command fails to grep for the test server it will return an exit code of `1` which will naturally trigger a failure without these additional conditions.

```
$ ansible localhost -m shell -a "echo 'foo' | grep bar"

localhost | FAILED | rc=1 >>
non-zero return code
```

File: tasks/server_setup.yml (hunk 2 above)
Comment (johnduarte) [inline tasks/server_setup.yml]: The openstack cli offers an easier way to get the status value:
```
openstack server show -f value -c status \"{{ test_server }}\"
```

File: tasks/server_setup.yml [hunk 3]
```diff
@@ -10,3 +10,24 @@
...
+  shell: |
+    lxc-attach -n "{{ utility_container.stdout }}"  \
+    -- bash -c '. /root/openrc ; \
+    openstack server list | grep "{{ test_server }}"'
+  register: test_server_instance
+  changed_when: false
+  failed_when: false
+
+- name: Check test server status
+  # helpers.get_expected_value('server', instance_name, 'status', 'ACTIVE', host))
```
Comment (ryan-rs) [inline tasks/server_setup.yml]: Why is this line here?

Comment (phongdly) [reply tasks/server_setup.yml]: The line is like documenting the action in the ansible task, I have no problem to remove it.

**Comment by phongdly** [comment]
@johnduarte , @ryan-rs : updated!
